lib/hpc/utils.pm

```diff
@@ -21,11 +21,13 @@ our @EXPORT = qw(get_slurm_version);
 sub get_mpi() {
     my $mpi = get_required_var('MPI');
 
-    if ($mpi eq 'openmpi3') {
+    if ($mpi eq 'openmpi') {
         if (is_sle('<15')) {
-            $mpi = 'openmpi';
+            $mpi = 'openmpi1';
         } elsif (is_sle('<15-SP2')) {
             $mpi = 'openmpi2';
+        } else {
+            $mpi = 'openmpi3';
```

Comment:
I would completely  remove whole `if-if-else` section  and just rely on defined variable at job group level. These conditions made sense in the past when we couldn't set variable at test suite level per different product, but we can  do it now.

Reply:
my intention was to keep it at first and then working on unifying all hpc test using yaml, so we could further simplify if we find it useful.
If I'm correct in current state, where we have limited OSD mpi test definitions, we would need to have multiple mpi test suits in OSD.

Reply:
If HPC had proper yaml configuration, there would be only one generic test suite, but yes, we are not there yet.
So let;s simplify it in code first and we can do another refactoring while we fully use yaml defintion.